### PR TITLE
[add] Profile Update Logic

### PR DESCRIPTION
**Pull request by edbella**

### CODE REVIEW

File: src/components/vendorDashboard/components/profile/components/ProfileData.jsx [hunk 1]
```diff
@@ -20,10 +21,13 @@ const {
  * @function
  * @param {object} props - userObj being passed as prop
  */
-function DisplayDetails() {
+function DisplayDetails(props) {
+    const { userObj, userType } = props;
     return (
         <Col xs={24} md={12} lg={9}>
-            <Empty description={NO_DATA} image={Empty.PRESENTED_IMAGE_SIMPLE} />
+            {userObj ? (
```
Comment (Way2nnadi) [inline src/components/vendorDashboard/components/profile/components/ProfileData.jsx]: users an object as a conditional can lead to buggy behavior. What happens if you get an object but it is empty?

Comment (Way2nnadi) [reply src/components/vendorDashboard/components/profile/components/ProfileData.jsx]: Use a boolean for conditional checks

File: src/components/vendorDashboard/components/profile/sagas.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+import { auth } from '../../../../firebase';
+
+const API_URL = process.env.REACT_APP_PROFILE_API;
+
+// Destructuring actionTYpes
+const {
+    UPDATE_PROFILE, UPDATE_SUCCESS,
+} = actionTypes;
+
+// Destructuring Actions
```
Comment (Way2nnadi) [inline src/components/vendorDashboard/components/profile/sagas.js]: these comments are not really needed, but if the help you think about the code then go ahead and keep adding them

File: src/components/vendorDashboard/components/profile/components/ProfileData.jsx (hunk 1 above)
Comment (edbella) [reply src/components/vendorDashboard/components/profile/components/ProfileData.jsx]: Ok. Thanks

**Pull request merged**